Consistent use of "private" vs "internal" terms for GitHub Apps (#6030)
github/docs: content/developers/apps/making-a-github-app-public-or-private.md

### Private installation flow

> Private installation flows allow only the owner of a GitHub App to install it. Limited information about the GitHub App will still exist on a public page, but the **Install** button will only be available to organization administrators or the user account if the GitHub App is owned by an individual account. Private GitHub Apps can only be installed on the user or organization account of the owner.

Reviewer: ```suggestion
Private installation flows allow only the owner of a GitHub App to install it. Limited information about the GitHub App will still exist on a public page, but the **Install** button will only be available to organization administrators or the user account if the GitHub App is owned by an individual account. Private{% if currentVersion ver_lt "enterprise-server@3.2" or currentVersion == "github-ae@latest" %}, or internal,{% endif %} GitHub Apps can only be installed on the user or organization account of the owner.
```

3. Select the GitHub App whose installation option you want to change.
![App selection](/assets/images/github-apps/github_apps_select-app.png)
{% data reusables.user-settings.github_apps_advanced %}

> 5. Depending on the installation option of your GitHub App, click either **Make public** or **Make private**.

Reviewer: ```suggestion
5. Depending on the installation option of your GitHub App, click either **Make public** or **Make private{% if currentVersion ver_lt "enterprise-server@3.2" or currentVersion == "github-ae@latest" %}internal{% endif %}**.
```

> 5. Depending on the installation option of your GitHub App, click either **Make public** or **Make {% if currentVersion == "free-pro-team@latest" %}private{% else %}internal{% endif %}**.

Reviewer: ```suggestion
5. Depending on the installation option of your GitHub App, click either **Make public** or **Make {% if currentVersion == "free-pro-team@latest" or currentVersion ver_gt "enterprise-server@3.1" %}private{% else %}internal{% endif %}**.
```

> 5. Depending on the installation option of your GitHub App, click either **Make public** or **Make private{% if currentVersion ver_lt "enterprise-server@3.2" or currentVersion == "github-ae@latest" %}internal{% endif %}**.

Reviewer: ```suggestion
5. Depending on the installation option of your GitHub App, click either **Make public** or **Make {% if currentVersion == "free-pro-team@latest" %}private{% else %}internal{% endif %}**.
```

![Button to change the installation option of your GitHub App](/assets/images/github-apps/github_apps_make_public.png)

> 6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App private**.

Reviewer: ```suggestion
6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App private{% if currentVersion ver_lt "enterprise-server@3.2" or currentVersion == "github-ae@latest" %}internal{% endif %}**.
```

> 6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App {% if currentVersion == "free-pro-team@latest" %}private{% else %}internal{% endif %}**.

Reviewer: ```suggestion
6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App {% if currentVersion == "free-pro-team@latest" or currentVersion ver_gt "enterprise-server@3.1" %}private{% else %}internal{% endif %}**.
```

> 6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App private{% if currentVersion ver_lt "enterprise-server@3.2" or currentVersion == "github-ae@latest" %}internal{% endif %}**.

Reviewer: ```suggestion
6. Depending on the installation option of your GitHub App, click either **Yes, make this GitHub App public** or **Yes, make this GitHub App {% if currentVersion == "free-pro-team@latest" %}private{% else %}internal{% endif %}**.
```
